jvb/src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -235,24 +226,9 @@ public Conference(Videobridge videobridge,
                     try
                     {
                         long start = System.currentTimeMillis();
-                        IQ requestIQ = request.getRequest();
-                        IQ response;
-                        boolean expire = false;
-                        if (requestIQ instanceof ColibriConferenceIQ)
-                        {
-                            response = shim.handleColibriConferenceIQ((ColibriConferenceIQ)requestIQ);
-                        }
-                        else if (requestIQ instanceof ConferenceModifyIQ)
-                        {
-                            Pair<IQ, Boolean> p =
-                                colibri2Handler.handleConferenceModifyIQ((ConferenceModifyIQ)requestIQ);
-                            response = p.getFirst();
-                            expire = p.getSecond();
-                        }
-                        else
-                        {
-                            throw new IllegalStateException("Bad IQ " + request.getClass() + " passed to colibriIQ");
-                        }
+                        Pair<IQ, Boolean> p = colibri2Handler.handleConferenceModifyIQ(request.getRequest());
+                        IQ response = p.getFirst();
+                        boolean expire = p.getSecond();
```

Comment:
Is a correct exception still thrown, and thus an XMPP error returned, for an unexpected IQ type?